src/reliability.jl

```diff
@@ -0,0 +1,78 @@
+using LinearAlgebra: isposdef
```

Comment:
Given that this is at the top level, it will affect other files, so better put it in src/StatsBase.jl.

Reply:
Done.